docs/myzrok/upgrading.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Upgrading From 0.4 to 1.0
+---
+
+## Upgrading an existing 0.4 environment
+If you have not already, [install the latest 1.x zrok binary](getting-started.mdx#installing-the-zrok-command) into your environment.
+
+Run the following to rebase your environment to use the new versioned API:
+```
+  zrok rebase apiEndpoint https://api-v1.zrok.io
+```
```

Comment:
This is no longer necessary as of `v1.0.2`, which includes automatic rebasing.

Reply:
I'll update the doc with the Note block indicating that as of 1.0.2, the rebase is automatic

Reply:
I would remove that entirely. Just upgrading to the latest `v1` binary will get them auto-rebase. Including those instructions would just be confusing.